New blog: A new 'Privacy Shield' won't solve big tech's GDPR problem (#2809)
PostHog/posthog.com: contents/blog/gdpr-google-analytics-privacy-shield.md

There's lots of detail in the [full ruling](https://noyb.eu/sites/default/files/2022-01/E-DSB%20-%20Google%20Analytics_EN_bk.pdf), but here's a quick summary:

- An Austrian website was found to have exported personal data of a user to Google servers in the US
- The complaint argued this was a breach as it was subject to demands from local intelligence agencies
- The Austrian Data Protection Agency agreed it was a breach and the case remains ongoing 

That's a very simplified version of events, but here are three important takeaways:

### 1 . Google itself was cleared of any wrongdoing
Only the website that was using Google Analytics, as the data controller, was found liable for the breach. Under the GDPR, breaching the regulation can result in huge fines of up to €20 million or 4% of a company's global turnover, whichever is the largest. [Amazon was fined €746 million by authorities in Luxembourg in 2020](https://www.bbc.co.uk/news/business-58024116), the largest GDPR fine so far. Getting it wrong can be painful.

> ### 2. Any website using GA in the EU could be liable

Reviewer: Does this mean US websites with EU visitors? 

What do you specifically mean by "using Google Analytics in the EU"?

Author: Strictly speaking, it could mean any website using GA who collects data on EU citizens, so perhaps 'Any website using GA could be liable' is more accurate?

Reviewer: could we just rephrase to that then? Ie "any website using GA, collecting data on EU citizens"

Reviewer: (generally, being a bit more specific as long as we're correct is a good thing - our audience love the actual details where we can provide them without legal risk!)

---
date: 2022-01-24
title: A new 'Privacy Shield' won't solve big tech's GDPR problem
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: ../images/blog/posthog-release-notes-blog.png
featuredImageType: full
author: ["andy-vandervell"]
categories: ["Privacy", "Product Analytics", "Perspectives", "General"]
---

> Ten years ago today, the European Commission published the first draft of the General Data Protection Regulation (GDPR). But, as the recent ruling in Austria concerning one website's use of Google Analytics proves, how those rules are enforced and interpreted is still contested.

Reviewer 2: ```suggestion
Ten years ago today, the European Commission published the first draft of the General Data Protection Regulation (GDPR). But, as [the recent ruling in Austria concerning one website's use of Google Analytics](https://isgoogleanalyticsillegal.com/) proves, how those rules are enforced and interpreted is still contested.
```

> There's lots of detail in the full ruling (link needed), but here's a quick TL;DR: 

Reviewer 2: ```suggestion
There's lots of detail in the full ruling (link needed), but here's a quick summary:
```

Would be my preference

- An Austrian website was found to have exported personal data of a user to Google servers in the US
- The complaint argued this was a breach as it was subject to demands from local intelligence agencies
- The Austrian Data Protection Agency agreed it was a breach and the case remains ongoing 

That's a very simplified version of events, but here are three important takeaways:

> ### 1 . Google was cleared of any wrongdoing (for now)

Reviewer 3: Google 'itself' was cleared... I'd say for emphasis and remove (for now), it sounds salacious/speculative

> Only the website that was using Google Analytics, as the data controller, was found liable for the breach. Under the GDPR, breaching the regulation can result in huge fines of up to €20 million or 4% of a company's global turnover, whichever is the largest. Amazon was fined €746 million by authorities in Luxembourg in 2020, the largest GDPR fine so far. Getting it wrong can be painful.

Reviewer 2: ```suggestion
Only the website that was using Google Analytics, as the data controller, was found liable for the breach. Under the GDPR, breaching the regulation can result in huge fines of up to €20 million or 4% of a company's global turnover, whichever is the largest. [Amazon was fined €746 million by authorities in Luxembourg in 2020](https://www.bbc.co.uk/news/business-58024116), the largest GDPR fine so far. Getting it wrong can be painful.
```

### 3. This is just the beginning
This complaint was but one of 100 similar complaints lodged by the same non-profit group. The founder, Max Schrems, [told WIRED](https://www.wired.co.uk/article/google-analytics-europe-austria-privacy-shield): "It's not specific to Google Analytics. It's basically about outsourcing to US providers in general". Other complaints include the use of Google Analytics and Facebook Connect by the likes of Airbnb and Ikea, so expect to hear about more rulings in the coming months.

## How did we get here? A brief history of GDPR
Most of the conversation thus far has focused on whether this means [Google Analytics is illegal](https://isgoogleanalyticsillegal.com/) to use in Europe, but this goes way beyond Google Analytics alone. This is just the latest chapter in a long running saga concerning a core principle of the GDPR: data portability.

In its [response](https://blog.google/around-the-globe/google-europe/its-time-for-a-new-eu-us-data-transfer-framework/) to the ruling, Google urged the EU and US to come up with a new legal framework to create legal stability. It's a reasonable demand but, we've been here several times before.

> To understand the problem a little better, it helps to understand the history of GDPR. Let's break things down: 

Reviewer 3: Any chance you can get this down to 10 - 'a history of GDPR in 10 lessons' or something

### June 2013: Edward Snowden leaks the largest cache of intelligence data in history
Edward Snowden leaked millions of secret US intelligence files detailing the huge extent of digital surveillance. This leak fundamentally changed the debate around personal privacy and data protection forever, and it's an important factor in much of what's happened since.

### October 2015: European court deems EU-US data protection agreement 'invalid'
Responding to a complaint from privacy campaigner Max Schrems, the European Court of Justice deemed the 'Safe Harbor Privacy Principles' allowing data portability between the EU and US invalid. Schrems used the Snowden revelations, arguing that data transferred by Facebook from Ireland to US servers was insufficiently protected due to the NSA's surveillance practices. [The ruling](https://iapp.org/resources/article/schrems-i/) became known as 'Schrems I'.

> ### May 24, 2016: The GDPR comes is approved by EU

Reviewer 2: ```suggestion
### May 24, 2016: GDPR is approved by EU
```

Not sure what comes is means

### August 17, 2020: Max Schrems files complaint with Austrian DPA
Schrems alleges an Austrian website and Google violated the GDPR when a data subject visited a website on health topics. The complaint details how personal data, including user identifiers, IP address and browser details, were exported from Austria to the US where it could be subject to local law enforcement requests for access.

### July 16, 2021: Amazon fined €746 million for GDPR breach
Amazon [is fined](https://www.wired.co.uk/article/amazon-gdpr-fine) €746 million for failing to follow rules regarding its use of targeted advertising and proper consent. It's the largest GDPR fine to date.

### January 13, 2022: Austrian authority rules Google Analytics violates GDPR
The Austrian data authority partially upholds the complaint, ruling that the Austrian website did violate the GDPR. Google, as a data processor, isn't deemed liable.

> ## What happens next?

Reviewer 3: This would be a great place to introduce a link and screenshot to isgoogleanalyticsillegal.com potentially?

> Amazon [is fined](https://www.wired.co.uk/article/amazon-gdpr-fine) nearly $900 million for failing to follow rules regarding its use of targeted advertising and proper consent. It's the largest GDPR fine to date.

Reviewer 2: ```suggestion
Amazon [is fined](https://www.wired.co.uk/article/amazon-gdpr-fine) €746 million for failing to follow rules regarding its use of targeted advertising and proper consent. It's the largest GDPR fine to date.
```

Making the figure accurate and consistent with the denomination of the previous reference.
